app/models/claim/search.rb

```diff
@@ -0,0 +1,31 @@
+class Claim
+  class Search
+    attr_accessor :query
```

Comment:
"query" is quite a loaded term. Can we rename this to something less ambiguous? Maybe "search_term" or "search_string"?